### PR TITLE
docs(orders): ORDERS-4516 Add support for pickup_method_id in pickup …

### PR DESCRIPTION
**Pull request by dornah**
…consignments.

### What & Why
Update the api spec to reflect the changes made in https://github.com/bigcommerce/bigcommerce/pull/43865 regarding using pickup_method_id in orders api.

### Proof
Screen Shot 2021-12-16 at 9 30 01 am
Screen Shot 2021-12-16 at 9 32 36 am
Screen Shot 2021-12-16 at 9 33 43 am
Screen Shot 2021-12-16 at 9 34 45 am

### CODE REVIEW

File: reference/orders.v2.oas2.yml [hunk 1]
```diff
@@ -4768,18 +4772,38 @@ components:
...
+          example: 5
+        pickup_method_display_name:
+          type: string
+          description: A name for the pickup method that can be displayed to shoppers.
+          maxLength: 250
+          example: Pick Up
+        collection_instructions:
+          type: string
+          description: A message for shoppers explaining how to collect their pickup order.
+          maxLength: 250
```
Comment (WillemHoman) [inline reference/orders.v2.oas2.yml:4791]: did we end up sticking with 250 for this?

**Review by WillemHoman — APPROVED** [review]

File: reference/orders.v2.oas2.yml (hunk 1 above)
Comment (donald-nguyen-bc) [inline reference/orders.v2.oas2.yml:4791]: should we change this to bigger number as discussed so we don't have to update the doc again ?

**Review by donald-nguyen-bc — APPROVED** [review]
👍

File: reference/orders.v2.oas2.yml (hunk 1 above)
Comment (dornah) [reply reference/orders.v2.oas2.yml:4791]: This is the current implementation. So I figured make it reflect the reality and we can update the specs if we change it to text.
Also, it's easy to increase the max length in the future if we decide to do so. I don't have a strong preference for this one, so happy with doing it now as well if the team prefers it.

Comment (dornah) [reply reference/orders.v2.oas2.yml:4791]: Although, 250 does sound small. So I'll create a ticket for it.
I just checked the letters for the Direct to Boot description of woolworth in https://www.woolworths.com.au/shop/discover/shopping-online/pickup, and it's 519.

Comment (dornah) [reply reference/orders.v2.oas2.yml:4791]: [ORDERS-4542](https://jira.bigcommerce.com/browse/ORDERS-4542)

Comment (dornah) [reply reference/orders.v2.oas2.yml]: Removed the max_length for collection_instructions for now. In ORDERS-4542 we can decide what limit we want to set for the user and add it here.

**Pull request merged**